.github/workflows/ci-workflow.yml

```diff
@@ -44,11 +40,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -e .[dev]
-    - name: Lint with flake8
-      run: |
-        flake8 csp_billing_adapter
-        flake8 tests
-    - name: Test with pytest
+        pip install tox tox-gh-actions
```

Comment:
Have we validated that tox-gh-actions is a safe package to use?

Reply:
It is MIT Licensed - checked that when I leveraged it for scc-hypervisor-collector last year.

Beyond that I didn't dig too deeply into it.

Reply:
As mentioned above, if we want to be more explicit and run each each test specifically, e.g. `tox -e check` and `tox -e pytest`, then we don't need the tox-gh-actions plugin.

Reply:
Have updated the GitHub Actions to explicitly specify which test env to run with tox, eliminating the need for tox-gh-actions.